src/DGtal/io/viewers/Viewer3D.ih

```diff
@@ -820,7 +820,7 @@ DGtal::Viewer3D<Space, KSpace>::init()
   myGLScaleFactorY=1.0;
   myGLScaleFactorZ=1.0;
   myGLLineMinWidth=1.5;
-
+  setBackgroundColor ( QColor ( 217, 228, 255 ) );
```

Comment:
For the color ok but the dark one is no more present. 
We can perhaps put the new one by default (instead white) and let the dark one associated to Key_B ?